Sources/DiscordModels/Types/Gateway.swift

```diff
@@ -477,6 +477,18 @@ public struct Gateway: Sendable, Codable {
                 self.status = status
                 self.afk = afk
             }
+
+            public func encode(to encoder: any Encoder) throws {
+                var container = encoder.container(keyedBy: CodingKeys.self)
+                if let since {
+                    try container.encode(self.since, forKey: .since)
```

Comment:
This one shouldn't be self.since, just since 🙂

Reply:
omg... I need to have some rest =) fixed... thx!

Reply:
no worries :)